### ISSUE TITLE
[Crash]: Server + Client Crash when Entering Nether [URGENT[

### ISSUE THREAD

**Issue by Splurpy** [CorgiTaco/BYG#721]
### Minecraft Version

1.16.5

### BYG Versions

1.3.5

### Terrablender

N/A

### Mod Loader

Forge

### Mod Loader Version

36.2.20

### What happened?

No matter what biome I'm in (Vanilla or BYG), the game crashes when attempting to load the nether.
This problem doesn't persist with the End.

I am playing with mods, but no new mods have been added since I was last able to enter the nether.

### crash-log

[crash-2022-04-20_13.37.58-server.txt](https://github.com/AOCAWOL/BYG/files/8523920/crash-2022-04-20_13.37.58-server.txt)

### latest.log

[latest.log](https://github.com/AOCAWOL/BYG/files/8523923/latest.log)

**Comment by Splurpy** [comment]
The issue also doesn't persist when BYG is removed from the Modpack, so I believe it's a conflict. I'm checking through all of my mods to see now.

Confirmed: The issue is with BYG. I removed every last single mod, including optifine, and the issue persists on new worlds and old worlds alike.

I have a server with friends running BYG, and it's going to be really unfortunate if we can't enter the nether. I didn't think this would be an issue.
